org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -66,15 +66,11 @@ Parameters:
     Type: String
     Default: '906769aa66-4c409d3e-e03d-4f46-aa24-840c977ef9d4'
 
-  workflowNextflowDevViewerGroup:
-    Type: String
-    Default: '906769aa66-eae974c2-3cad-4477-926a-17a2c7ed8e81'
-
   workflowNextflowProdAdminGroup:   #JC aws-workflow-nextflow-prod-admin
     Type: String
     Default: '906769aa66-5c67ab0f-1091-48f9-b5b9-37d8b99a446a'
 
-  workflowNextflowProdViewerGroup:
+  workflowNextflowTowerViewerGroup:   #JC aws-workflow-nextflow-tower-viewer
     Type: String
     Default: '906769aa66-6cdd2cc1-6d3d-49d8-b105-37c22bb3d402'
```

Comment:
I've reconfigured JC with new group which created new SSO group therefore you'll need to update this to `906769aa66-6cdd2cc1-6d3d-49d8-b105-37c22bb3d402`

Reply:
oops, i double checked and it is indeed the same.